### ISSUE TITLE
[Bug]: termux-tools pkg command

### ISSUE THREAD

**Issue by Sarisan** [termux/termux-packages#10460]
### Problem description

termux-tools installed from pacman uses apt in pkg.

### What steps will reproduce the bug?

Install termux-tools from pacman and try to install any package with pkg.

### What is the expected behavior?

It should use pacman as it is installed from it.

### System information

termux-info:

```
Termux Variables:
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PID=4467
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=pacman
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# /data/data/com.termux/files/usr/etc/pacman.conf
[main]
Server = https://s3.amazonaws.com/termux-main.pacman/aarch64
[x11]
Server = https://s3.amazonaws.com/termux-x11.pacman/aarch64
[root]
Server = https://s3.amazonaws.com/termux-root.pacman/aarch64
Updatable packages:
All packages up to date
Android version:
11
Kernel build information:
Linux localhost 4.14.245-QuicksilveR-ginkgo-v2.5/a2dbd412 #1 SMP PREEMPT Mon Oct 4 02:10:01 CEST 2021 aarch64 Android
Device manufacturer:
Xiaomi
Device model:
Redmi Note 8
```

**Comment by 2096779623** [comment]
The pacman source is currently not mirrored, so the termux-change-repo is not required, and the termux-tools package needs to be modified.
@Maxython

**Comment by thunder-coding** [comment]
Fixed in #10486

**Comment by mtnnvnk7-sudo** [comment]
https://github.com/termux/termux-packages/issues/10460
